docs/usage.md

```diff
@@ -1,12 +1,19 @@
 # Usage
 
-## Component Project
+The [C#/WinRT NuGet package](https://www.nuget.org/packages/Microsoft.Windows.CsWinRT/) provides tooling for the following scenarios:
 
-A component project adds a NuGet reference to C#/WinRT to invoke cswinrt.exe at build time, generate projection sources, and compile these into an interop assembly. For an example of this, see the [Projection Sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/Net5ProjectionSample). Command line options can be displayed by running **cswinrt -?**.  The interop assembly is then typically distributed as a NuGet package itself. 
+- [Generate and distribute an interop assembly](#generate-and-distribute-an-interop-assembly)
+- [Author and consume a C#/WinRT component](#author-and-consume-a-cwinrt-component)
```

Comment:
Does this link work as expected for you? For me it goes to a page that is failing to render the mark down properly

Reply:
yes, the anchor tag includes "cwinrt" - the "#/" probably needs to be escaped

Reply:
the link works for me